### PR TITLE
Add source installation documentation

### PR DESCRIPTION
**Pull request by jparisu**
Signed-off-by: jparisu <javierparis@eprosima.com>

### CODE REVIEW

File: docs/rst/developer_manual/installation/sources/linux.rst [hunk 1]
```diff
@@ -1,7 +1,326 @@
...
+            -DCMAKE_INSTALL_PREFIX=~/Fast-DDS-Monitor/install \
+            -DCMAKE_PREFIX_PATH=~/Fast-DDS-Monitor/install \
+            -DQT_PATH=<qt-installation-path>
+        cmake --build . --target install
+
+
+.. note::
+
+    By default, *eProsima Fast DDS Monitor* does not compile tests.
+    However, they can be activated by downloading and installing `Gtest <https://github.com/google/googletest>`_.
```
Comment (jparisu) [inline docs/rst/developer_manual/installation/sources/linux.rst:285]:
```suggestion
    However, they can be activated by downloading and installing `Gtest <https://github.com/google/googletest>`_ and building with CMake option `-DBUILD_TESTS=ON`.
```

File: docs/rst/developer_manual/installation/sources/linux.rst [hunk 2]
```diff
@@ -1,7 +1,326 @@
...
+
+To run the *eProsima Fast DDS Monitor* application, source the *Fast DDS* and *Fast DDS Statistics Backend* libraries
+and execute the executable file that has been installed in :code:`<install-path>/fastdds_monitor/bin/fastdds_monitor`:
+
+.. code-block:: bash
+
+    source install/setup.bash
+    ./<install-path>/fastdds_monitor/bin/fastdds_monitor
+
+Be sure that this executable has executable permissions.
```
Comment (jparisu) [inline docs/rst/developer_manual/installation/sources/linux.rst:315]:
```suggestion
Be sure that this executable has execution permissions.
```
??

File: docs/rst/developer_manual/installation/sources/linux.rst [hunk 3]
```diff
@@ -1,7 +1,326 @@
...
+
+Run an application
+==================
+
+To run the *eProsima Fast DDS Monitor* application, source the *Fast DDS* and *Fast DDS Statistics Backend* libraries
+and execute the executable file that has been installed in :code:`<install-path>/fastdds_monitor/bin/fastdds_monitor`:
+
+.. code-block:: bash
+
+    source install/setup.bash
```
Comment (jparisu) [inline docs/rst/developer_manual/installation/sources/linux.rst:312]:
```suggestion
    # if built has been done by colcon, all projects could be sourced like this
    source install/setup.bash  
```

File: docs/rst/developer_manual/installation/sources/linux.rst [hunk 4]
```diff
@@ -1,7 +1,326 @@
...
+Qt 5.15 is needed in order to build *Fast DDS Monitor*.
+To install this Qt version, refer to `Qt Downloads <https://www.qt.io/download>`_ website.
+
+
+.. _colcon_installation:
+
+Colcon installation
+===================
+
+Create a :code:`Fast-DDS-Monitor` directory and download the :code:`.repos` file that will be used to install
```
Comment (jparisu) [inline docs/rst/developer_manual/installation/sources/linux.rst:167]:
```suggestion
#. Create a :code:`Fast-DDS-Monitor` directory and download the :code:`.repos` file that will be used to install
```

File: docs/rst/developer_manual/installation/sources/linux.rst [hunk 5]
```diff
@@ -1,7 +1,326 @@
...
+For more details, please refer to the :ref:`cmake_options` section.
+For a detailed description of the Gtest installation process, please refer to the
+`Gtest Installation Guide <https://github.com/google/googletest>`_.
+
+.. _dependencies:
+
+Dependencies
+------------
+
+*eProsima Fast DDS* has the following dependencies, when installed from binaries in a Linux environment:
```
Comment (jparisu) [inline docs/rst/developer_manual/installation/sources/linux.rst:101]: I would clear out that this is only needed if Fast DDS is not yet installed

**Review by jparisu — COMMENTED** [review]
LGTM

**Review by rsanchez15 — DISMISSED** [review]
Approved in favour of https://github.com/eProsima/Fast-DDS-monitor/pull/60#pullrequestreview-684846961

**Review by rsanchez15 — APPROVED** [review]
Approved in favour of https://github.com/eProsima/Fast-DDS-monitor/pull/60#pullrequestreview-684846961

**Pull request merged**